File: Student_Attendance_Record_I.cpp
class Solution {
public:
    bool checkRecord(string s) {
        int a = 0 , l = 0;
        for(auto i : s){
            if(i == 'A') a++;
            if(i == 'L') l++;
            else l = 0;
            if(a >= 2 || l > 2) return false;
        }
        return true;
    }
};

File: Student_Attendance_Record_I.java
class Solution {
    public boolean checkRecord(String s) {
        int a=0;
        int l=0;
        for(int i=0;i<s.length();i++){
            if(a==2) return false;
            if(l==3) return false;
            if(s.charAt(i)=='A') a++;
            if(s.charAt(i)=='L')l++;
            else l=0;
            }
            if(a>=2) return false;
            if(l>=3) return false;
            return true;
            }
    }
